include/MixerChannelView.h

```diff
@@ -141,4 +137,4 @@ namespace lmms::gui
     };
 } // namespace lmms::gui
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
```suggestion
#endif // MIXER_CHANNEL_VIEW_H
```
Not really necessary but would be nice.